File: ruby/src/careteam.rb
require './fhir_client'

class CareTeamClient
  def get_careteam(id)
    FhirClient.new

    FHIR::CareTeam.read(id)
  end

  def get_careteam_with_practitioners(id)
    FhirClient.new

    params = {
      '_id': id,
      '_include': 'CareTeam:participant'
    }
    FHIR::CareTeam.search(params)
  end

end

File: javascript/js/careteam.js
import Client from "./client";

class CareTeamClient {
  async get() {
    const client = new Client();
    return client.request('CareTeam/1');
  }

  async getWithPractitioners() {
    const client = new Client();
    return client.request('CareTeam/?_id=1&_include=CareTeam:participant');
  }
}

export default CareTeamClient;
